lib/api/core/validation/types/numeric.js

```diff
@@ -67,19 +74,36 @@ class NumericType extends BaseType {
    * @throws {PreconditionError}
    */
   validateFieldSpecification(typeOptions) {
-    if (typeOptions.hasOwnProperty('range')) {
+    if (Object.prototype.hasOwnProperty.call(typeOptions, 'range')) {
       if (!this.checkAllowedProperties(typeOptions.range, ['min', 'max'])) {
-        throw new PreconditionError('Invalid "range" option definition');
+        errorsManager.throw('internal', 'validation', 'invalid_range_option');
       }
 
       for (const prop of ['min', 'max']) {
+<<<<<<< HEAD
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Parse error at line 83 column 2:

73:    * @return {TypeOptions}
74:    * @throws {PreconditionError}
75:    */
76:   validateFieldSpecification(typeOptions) {
77:     if (Object.prototype.hasOwnProperty.call(typeOptions, 'range')) {
78:       if (!this.checkAllowedProperties(typeOptions.range, ['min', 'max'])) {
79:         errorsManager.throw('internal', 'validation', 'invalid_range_option');
80:       }
81: 
82:       for (const prop of ['min', 'max']) {
83: <<<<<<< HEAD
     ^
84:         if (typeOptions.range.hasOwnProperty(prop) && typeof typeOptions.range[prop] !== 'number') {
85:           errorsManager.throw('internal', 'validation', 'invalid_range_type', prop);
86:         }
87:       }
88: 
89:       if (typeOptions.range.hasOwnProperty('min') && typeOptions.range.hasOwnProperty('max') && typeOptions.range.max < typeOptions.range.min) {
90:         errorsManager.throw('internal', 'validation', 'invalid_range');
91: =======
92:         if (
93:           Object.prototype.hasOwnProperty.call(typeOptions.range, prop) [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AParsingError)